src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreServiceCollectionExtensions.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.AspNetCore.Mvc.ApplicationModels;
 using Microsoft.AspNetCore.Mvc.ApplicationParts;
 using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.AspNetCore.Mvc.ErrorDescription;
```

Comment:
Can we not have this?

Reply:
Sure. `.Infrastructure` namespace? It doesn't seem top-level worthy.